src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -598,14 +601,35 @@ int approximateNodeCount() {
         return iota.get();
     }
 
+    /** For diagnostic purposes only. */
```

Comment:
So I personally appreciate it when diagnostics methods like this have javadoc or comments giving a quick explanation of what their diagnostics are meant to show - not a blocker by any means, just a casual request.